packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -89,6 +89,40 @@ export const makeBidSpecShape = (currencyBrand, collateralBrand) => {
 
 /** @typedef {import('@agoric/vat-data').Baggage} Baggage */
 
+/**
+ * @typedef {object} PriceLevel
```

Comment:
surprised we don't have this type already. closest is `PriceDescription` but that has more.

consider putting this in the `zoe/tools/types-ambient.js` since it makes sense outside of auctions